conda.yaml

```diff
@@ -1,22 +1,111 @@
 name: dreamcenter
 channels:
+  - conda-forge
```

Comment:
I needed to add conda-forge otherwise conda was not giving me a proper python3.10 installation

python3.10 is needed for `match` syntax